node/src/types/node_id.rs

```diff
@@ -84,7 +84,7 @@ impl DocExample for NodeId {
 
 impl Debug for NodeId {
     fn fmt(&self, formatter: &mut Formatter<'_>) -> fmt::Result {
-        write!(formatter, "NodeId({})", HexFmt(&self.0))
+        write!(formatter, "NodeId({})", base16::encode_lower(&self.0))
```

Comment:
```suggestion
        write!(formatter, "NodeId({})", checksummed_hex::encode(&self.0))
```

Reply:
I'll omit this suggestion too for consistency with `deserialize()` implementation.